app/main.py

```diff
@@ -1,3 +1,15 @@
 def create_dict(*args):
-    # write your code here
-    pass
+    dicto = {}
+    for index, arg in enumerate(args):
+        if isinstance(arg, (set, list, dict)):
+            print(f"Cannot add {arg} to the dict!")
+        elif not isinstance(arg, (set, list, dict)):
+            dicto[arg] = index
+        elif isinstance(arg, tuple):
+            for i in arg:
+                if isinstance(i, (set, list, dict)):
+                    print(f"Cannot add {arg} to the dict!")
+                    break
+                else:
+                    dicto[arg] = index
```

Comment:
imagine 'arg = (3, [1, 2], 5)', in loop, 1st iteration i = 3 and you add key: dicto[arg] = index, on 2nd iteration i = [1, 2] and you print that arg cannot ad to the dict. It's wrong